### DISCUSSION TITLE
Dehaydrate a Multi-Select Field

### DISCUSSION THREAD

**Discussion by BasharAl-Mohammad** [filamentphp/filament#4646] [Help] [answered]
Hello everyone,

I have this multi-select relationship field that I want to exclude from saving to the database.

![image](https://user-images.githubusercontent.com/55070053/196938171-e4e67d93-529d-42d9-a1a7-1e1d7d5b6612.png)

Tried `->dehydrated(false)` but still getting the data saved which I don't want. 

Here's the select code: 

```
Select::make('options')
    ->relationship('options','name')
    ->multiple()
    ->preload()
    ->dehydrated(false)
    ->createOptionForm([
        TextInput::make('name')
        ->unique(),
        ])
```

**Comment by danharrin** [comment, marked as answer]
`dehydrated(false)` is already called internally since the options are saved to a relationship and not the main model. So try `saveRelationshipsUsing(null)`

**Reply by BasharAl-Mohammad** [reply]
Works just perfectly.
Appreciate that a lot
